code/game/gamemodes/objective.dm

```diff
@@ -298,7 +298,11 @@ GLOBAL_LIST_EMPTY(objectives)
 	return target
 
 /datum/objective/protect/check_completion()
-	return !target || considered_alive(target, enforce_human = human_check)
+	var/obj/item/organ/brain/brain_target
+	if(human_check)
+		brain_target = target.current.getorganslot(ORGAN_SLOT_BRAIN)
+	//Protect will always suceed when someone suicides
+	return !target || considered_alive(target, enforce_human = human_check) || (human_check == TRUE && brain_target)? !brain_target.suicided : FALSE
```

Comment:
If brain_target.suicided = FALSE then that will be true and will always be completed, and if they suicide it will be false. Flip the !brain_target.suicided to just brain_target.suicided